grpc/sa-wrappers.go

```diff
@@ -1014,11 +1024,23 @@ func (sas StorageAuthorityServerWrapper) MarkCertificateRevoked(ctx context.Cont
 }
 
 func (sas StorageAuthorityServerWrapper) AddCertificate(ctx context.Context, request *sapb.AddCertificateRequest) (*sapb.AddCertificateResponse, error) {
+	// NOTE(@cpu): We allow `request.Issued` to be nil here for deployability aid.
+	// This allows a RA that hasn't been updated to send this parameter to operate
+	// correctly. We replace the nil value with a default in the SA's
+	// `AddCertificate` impl
 	if request == nil || request.Der == nil || request.RegID == nil {
 		return nil, errIncompleteRequest
 	}
 
-	digest, err := sas.inner.AddCertificate(ctx, request.Der, *request.RegID, request.Ocsp)
+	var issued *time.Time
+	// If the request.Issued int64 pointer isn't nil, create a pointer to
+	// a time.Time instance with its value.
+	if *request.Issued != 0 {
```

Comment:
This looks like it'll cause a nil pointer dereference when `request.Issued` is not set, should be `if request.Issued != nil` no? (also possibly `&& *request.Issued != 0`)

Reply:
Good catch :+1: